### ISSUE TITLE
Most recent Fabric update requires a non-existent version of Create: Fabric.

### ISSUE THREAD

**Issue by AttacMage** [sudolev/CreateInteriorsMod#23]
Unless I'm stupid and blind (likely), there is no 0.5.1h version of Create: Fabric (nor 0.5.1g, for that matter). 
Trying to use it with the currently available builds of Create causes a crash on bootup due to a dependency version mismatch.

**Comment by rhysdh540** [comment]
You're completely right, I depended on an in-development version of Create Fabric. If you know how to, you can setup a dependency override for Interiors (https://fabricmc.net/wiki/tutorial:dependency_overrides), but if not, you can use 0.5.4 for now.

**Comment by joernott** [comment]
From the "kids don't do that at home" section of my Minecraft config folder. This is, how my override in config/fabric_loader_dependencies.json looks:
```json
{
  "version": 1,
  "overrides": {
	"interiors": {
	  "-depends": {
		"create": "0.5.1-h-build.1516"
	  },
	  "+depends": {
		"create": "0.5.1-f-build.1417"
	  }
	}
  }
}
```

**Comment by joernott** [comment]
As create v0.5.1-j is out, this problem has solved itself.

**Comment by a0ax** [comment]
Good deal
